docs/first-order/classify_waveforms.ipynb

```diff
@@ -0,0 +1,2327 @@
+{
```

Comment:
<p>I suggest to re-shuffle a bit the paragraphs. In this paragraph you should be introducing the next bit of code, which is about sub-selecting cells. The thresholding is happening later on when you have the distribution. This is when the explanation would be useful. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/AllenInstitute/openscope_databook/pull/281/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>